.github/workflows/west-commands.yml

```
@@ -35,7 +45,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-24.04, macos-14, windows-latest]
+        os: [ubuntu-24.04, macos-15, windows-latest]
```

```suggestion
        os: [ubuntu-24.04, macos-14, windows-latest]
```
main is still on 14. llvm is on 15.